app/js/components/orderEntry/OrderHeader.jsx

```diff
@@ -1,13 +1,22 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import IconButton from '../button/IconButton';
 
 const OrderHeader = ({ status, orderable }) => (
   <Fragment>
     <th>{orderable}</th>
     <th>{status}</th>
-    <th>
-      <i className="icon-edit" />
-      <i className="icon-delete" />
+    <th className="draft-action-btns">
+      <IconButton
+        iconClass="icon-pencil"
+        iconTitle="EDIT"
+        onClick={() => {}}
+      />
+      <IconButton
+        iconClass="icon-remove"
+        iconTitle="DISCARD"
+        onClick={() => {}}
```

Comment:
`iconTitle="DISCARD"`, I think this is supposed to be `iconTitle="DISCONTINUE"`. Take a look at this snapshot from the previous implementation:
![screen shot 2018-08-17 at 11 37 42 am](https://user-images.githubusercontent.com/26048536/44262213-22196200-a212-11e8-8d95-bd29b3050294.png)